docker/deployment/scripts/kubeSyncMongo.sh

```diff
@@ -1,41 +1,39 @@
 #!/bin/bash
 
 set -e
+set -x
 
-echo Mongo Dump
-ssh mongo-xf mongodump
+PRODCONTEXT=--context='languageforge'
+QACONTEXT=--context='qa-languageforge'
+PRODSERVER=$(kubectl $PRODCONTEXT get pods --selector='app=db' -o name | sed -e s'/pod\///')
+QASERVER=$(kubectl $QACONTEXT get pods --selector='app=db' -o name | sed -e s'/pod\///')
 
-echo Remove Scripture Forge data from dump
-ssh mongo-xf rm -r dump/xforge
+echo "Mongo Dump on production"
+kubectl $PRODCONTEXT exec -c db $PRODSERVER -- bash -c "cd /data/db && mongodump"
```

Comment:
would you anticipate these dumps ever needing more space than might be available?